install_config/install/advanced_install.adoc

```diff
@@ -626,6 +626,63 @@ openshift_master_named_certificates=[{"certfile": "/root/STAR.openshift.com.crt"
 openshift_master_overwrite_named_certificates=true
 ----
 
+[[advanced-install-deployment-types]]
+=== Configuring Deployment Type
+
+Various defaults used throughout the playbooks and roles in this repository are
+set based on the deployment type configuration (usually defined in an Ansible
+hosts file).
+
+ifdef::openshift-enterprise[]
+Ensure the `openshift_deployment_type` parameter in your inventory file is set to
+`openshift-enterprise`.
+endif::[]
+ifdef::openshift-origin[]
+Ensure the `openshift_deployment_type` parameter in your inventory file is set to
+`origin`.
+endif::[]
+
+.Defaults for Deployment Type
```

Comment:
`deployment_type` / `openshift_deployment_type` for sure. The "defaults" part is the point for discussion.